requirements.txt

```diff
@@ -13,6 +13,8 @@ numpy
 wrapt
 cython
 tqdm
+filelock
```

Comment:
can't see any usage of `filelock` package, is it needed?

Reply:
Here: https://github.com/aimhubio/aim/blob/6fb8e1d1f31d233cf6541e8233ce847a8e7b010f/aim/storage/container.py#L1-L6

Reply:
my bad, have already marked the file as reviewed, that's why couldn't find the usage

Reply:
then we need to add those dependencies also in `setupy.py` as well, forgot to mention that